.github/workflows/main.yml

```diff
@@ -84,6 +84,8 @@ jobs:
 
       - name: CLI
         run: ./tests/e2e-test.sh
+      - name: PKG-CLI # this will a WIP to move all the CLI tests to the pkg repo
+        run: ./pkg/e2e-test.sh
```

Comment:
This actually refers to line 98 below, but github won't let me comment on it. Since you'll be generating a different code coverage file, you'll want to add it. If you follow my naming suggestion, that line will become:

`files: /tmp/rekor-merged.cov,/tmp/pkg-rekor-merged.cov`

Reply:
Thanks